gap/examples.gi

```diff
@@ -368,5 +368,91 @@ InstallMethod(GeneralisedPetersenGraph, "for a function, integer, integer",
 [IsFunction, IsInt, IsInt],
 GeneralisedPetersenGraphCons);
 
-InstallMethod(GeneralisedPetersenGraph, "for integer, integer", [IsInt, IsInt],
+InstallMethod(GeneralisedPetersenGraph, "for integer, integer",
```

Comment:
"for two integers" would be a better way to formulate this.

Reply:
Yes, sorry about that. I was trying to fix some linting related to line length and accidentally changed part of the Petersen Graph code. I will change it back to the way it was!